style.css

```diff
@@ -15,14 +15,17 @@ h1 {
   background-repeat: no-repeat;
   background-size: cover;
   background-position: center;
+  position: relative;
 }
     
-.hero-text {
-  text-align: center;
+.hero-inner {
+  border: 1px solid pink;
+  width: 100 px;
+  height: 100px;
```

Comment:
line 22, 23, 24 are obsolete and should be removed, we used them for visualisation purpose, and they affect how things work